modules/apps/collaboration/announcements/announcements-uad/bnd.bnd

```diff
@@ -1,3 +1,4 @@
 Bundle-Name: Liferay Announcements UAD
 Bundle-SymbolicName: com.liferay.announcements.uad
-Bundle-Version: 1.0.0
\ No newline at end of file
+Bundle-Version: 1.0.0
+-dsannotations-options: inherit
```

Comment:
What does `-dsannotations-options: inherit` do?

Reply:
It allows annotations on fields in base or abstract classes to be inherited by any subclasses.  In this case, it's necessary for the `@Reference` annotations in `BaseAnnouncementsUADEntityExporter` to be inherited and functioning by the Exporter classes that extend it.  Otherwise, you end up NPE's when those fields are called.

Reply:
There's some discussion that this should be on by default, but it hasn't happened yet.